presto-main/src/main/java/com/facebook/presto/server/security/ServerSecurityModule.java

```diff
@@ -66,6 +66,10 @@ private void setupLdap(LdapServerConfig ldapServerConfig, Binder binder)
             checkState(ldapServerConfig.getUserObjectClass() != null, "Missing property 'authentication.ldap.user-object-class'");
         }
 
+        if (ldapServerConfig.getLdapUrl().startsWith("ldap://")) {
```

Comment:
can you check if sth like `ldap.url=ldapserver:389` works? In this case it doesnt start with ldaps:// but is still using non-secure LDAP